kernel/build/config-arm64

```diff
@@ -2784,7 +2784,7 @@ CONFIG_IAVF=y
 CONFIG_I40EVF=y
 # CONFIG_ICE is not set
 # CONFIG_FM10K is not set
-# CONFIG_IGC is not set
+CONFIG_IGC=y
```

Comment:
do we need these NUC drivers for ARM?

Reply:
we tried to keep configs in sync across arches, if not something which can't be on a specific platform

I don't know if there's any ARM computer with that network card, but that's possible I guess?

Reply:
yeh, makes sense to keep both in sync